srcpkgs/sigil/template

```diff
@@ -1,30 +1,26 @@
 # Template file for 'sigil'
 pkgname=sigil
-version=1.9.10
-revision=4
+version=2.3.1
+revision=1
 build_style=cmake
-pycompile_dirs="usr/share/sigil/python3lib"
-# Can use system zlib, minizip and hunspell; can't use system pcre
 configure_args="-DUSE_SYSTEM_LIBS=ON"
-hostmakedepends="pkg-config unzip qt5-qmake"
-makedepends="zlib-devel qt5-tools-devel qt5-svg-devel
- qt5-webengine-devel qt5-xmlpatterns-devel qt5-multimedia-devel qt5-sensors-devel
- qt5-declarative-devel qt5-location-devel qt5-webchannel-devel
- qt5-plugin-odbc qt5-plugin-mysql qt5-plugin-pgsql qt5-plugin-sqlite qt5-plugin-tds
- boost-devel hunspell-devel python3-devel minizip-devel"
-depends="desktop-file-utils python3-lxml python3-css-parser python3-PyQt5-webengine"
+hostmakedepends="pkg-config qt6-tools"
+makedepends="qt6-base-devel qt6-webengine-devel qt6-svg-devel
+ python3-devel hunspell-devel pcre2-devel qt6-declarative-devel
+ qt6-base-private-devel minizip-devel"
 short_desc="Multi-platform EPUB ebook editor"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Emil Tomczyk <emru@emru.xyz>"
 license="GPL-3.0-or-later"
 homepage="https://github.com/Sigil-Ebook/Sigil"
 changelog="https://raw.githubusercontent.com/Sigil-Ebook/Sigil/master/ChangeLog.txt"
 distfiles="${homepage}/archive/${version}.tar.gz"
-checksum=35b6051d966c2848a8a031ebe691449229b3c95a56145647dc71857acc18fad5
+checksum=12e84d9a1fc41e60903557ea03801cd0004d8b75dfcebeb44c3a9b1fd8fa97ea
 python_version=3
 
+case "$XBPS_TARGET_MACHINE" in
+	armv5* | armv6* | armv7* | ppc* | riscv64* | i686*) broken="qt6-webengine missing" ;;
```

Comment:
Just depend on qt6-webengine and let the builder figure out it isn't buildable.

If it perpetually shows up in the list of build candidates, we can find a cleaner way to limit it.